### PR TITLE
Textfield v1 updates

### PR DESCRIPTION
**Pull request by samfloored**
@emma-floored 

Things here:
1. I added `header-text-fields` as a second class to apply to a text field. Happy to do this another way! It really mostly just clears out styles.
2. I moved text field helper messages to underneath the input like how errors are. Also moved the class to typography, where errors are.

screen shot 2019-02-15 at 5 39 30 pm
screen shot 2019-02-15 at 5 39 20 pm

Note: I think the colors aren't 100% final, still waiting on some design wisdom next week!

### CODE REVIEW

File: packages/blocks-base/styles/text-fields.styl [hunk 1]
```diff
@@ -5,14 +5,17 @@ $text-field-date-icon-color = $primary-00;
...
 
 $text-field-default-border = 1px solid $light-01;
 $text-field-active-border = 1px solid $light-02;
 $text-field-hover-border = 1px solid $light-02;
 $text-field-focus-border = 1px solid $focus;
 $text-field-error-border = 1px solid $danger-00;
 
+$header-text-field-background-color = $text-field-background-color;
+$header-text-field-invis-border = 1px solid transparent;
+$header-text-field-border = 1px solid #ced4d2;
```
Comment (emma-floored) [inline packages/blocks-base/styles/text-fields.styl]: oh I see, this is the new color you mentioned? the other 2 examples didn't look right?

File: packages/blocks-base/styles/typography.styl [hunk 2]
```diff
@@ -235,3 +235,11 @@ h5, .blx-h5 {
   position: absolute;
   top: 100%
 }
+
+.blx-input-helper-message {
+  blxUISmall()
+  color: $light-02;
+  position: absolute;
+  top: 100%;
```
Comment (emma-floored) [inline packages/blocks-base/styles/typography.styl]: just curious - what does this `top: 100%` do?

Comment (samfloored) [reply packages/blocks-base/styles/typography.styl]: It makes it show up underneath its parent (abs pos child of a rel pos parent, so the `100%` refers to the parent height) I think.

File: packages/blocks-base/styles/text-fields.styl (hunk 1 above)
Comment (samfloored) [reply packages/blocks-base/styles/text-fields.styl]: Yep! Sounds like it'll probably be `light-01`.

**Comment by emma-floored** [comment]
https://app.clubhouse.io/floored/story/13688/update-major-text-inputs

File: packages/blocks-docs/_includes/components/textfields.html [hunk 3]
```diff
@@ -124,7 +124,7 @@
   <div class="blx-text-field">
     <div class="blx-text-field-container">
       <input type="text" value="Text input">
-      <span class="blx-text-field-helper">34/150</span>
+      <span class="blx-input-helper-message">34/150</span>
```
Comment (emma-floored) [inline packages/blocks-docs/_includes/components/textfields.html]: might need to rename this class in the React component as well

Comment (samfloored) [reply packages/blocks-docs/_includes/components/textfields.html]: Is it possible it's not in any React component? Should I add something for it?

**Comment by samfloored** [comment]
@emma-floored I checked with Liz, and it sounds like we want the default (`light-01`) border. Updated, rebased, and fpushed. Any other thoughts on this?

**Pull request merged**